spec/form_builders/adp_text_field_spec.rb

```diff
@@ -160,7 +160,7 @@ def b100_with_value_with_hint
     context 'errored value with hint' do
       it 'produces error text' do
         resource.case_number = nil
-        resource.errors[:case_number] = 'Validation error here'
+        resource.errors.add(:case_number, 'Validation error here')
```

Comment:
this will need changing in the validation helpers to i imagine. which you may have done in later commits (just working through commit by commit atm)